metals/src/main/scala/scala/meta/internal/metals/StatusBarState.scala

```diff
@@ -0,0 +1,19 @@
+package scala.meta.internal.metals
+
+object StatusBarState {
+  sealed trait StatusBarState
+  case object On extends StatusBarState
+  case object Off extends StatusBarState
+  case object ShowMessage extends StatusBarState
+  case object LogMessage extends StatusBarState
+
+  def fromString(value: String): Option[StatusBarState] =
+    value match {
+      case "on" => Some(On)
+      case "Off" => Some(Off)
```

Comment:
ahh sorry I typed it instead of copy-paste it, it should be "off" and I capitalize it wrongly :facepalm:

Reply:
gosh, I didn't even notice this. Thanks for catching it! fixed!